wire-grpc-tests/src/test/proto-grpc/routeguide/FeatureDatabase.kt

```diff
@@ -0,0 +1,63 @@
+// Code generated by Wire protocol buffer compiler, do not edit.
+// Source file: routeguide/RouteGuideProto.proto
+package routeguide
```

Comment:
can you generate this?

Reply:
We do generate them but they're all checked in the codebase. Same is happening for current tests.
I'd rather do a follow up to clean the whole project than just us here.